docs/network_architecture.rst

```diff
@@ -348,15 +348,15 @@ where the request and response data are given as the following protobuf messages
    message StartTrainingRoundRequest {}
 
    message StartTrainingRoundResponse {
-       repeated xain_proto.numproto.NDArray weights = 1;
+       xain_proto.np.NDArray weights = 1;
```

Comment:
i understand this has been implemented elsewhere in this way (`xain_proto` repo), but i would advise against using the `np` abbreviation because that is a convention reserved for the actual `numpy` package. but it doesn't quite fall under the scope of this PR, i guess

Reply:
`np` was chosen explicitly as it stands for `numpy` (not `numproto`) to indicate directly that this is a protobuf implementation of numpy, as well as the associated utilites.